docs/api_reference/index.rst

```diff
@@ -6,7 +6,6 @@ API Reference
 .. toctree::
    :maxdepth: 1
 
-   fairlearn.datasets
```

Comment:
probably unintended?

Reply:
Intended, fairlearn.datasets is currently twice in the index (first and last).

Reply:
Ah, diff view doesn't show that!